src/main/java/net/minecrell/serverlistplus/core/player/ban/integration/AdvancedBanBanProvider.java

```diff
@@ -0,0 +1,57 @@
+package net.minecrell.serverlistplus.core.player.ban.integration;
+
+import me.leoko.advancedban.manager.PunishmentManager;
+import me.leoko.advancedban.utils.Punishment;
+import net.minecrell.serverlistplus.core.player.PlayerIdentity;
+import net.minecrell.serverlistplus.core.player.ban.BanProvider;
+
+import java.util.Date;
+
+public class AdvancedBanBanProvider implements BanProvider {
+
+    private String getUUID(PlayerIdentity playerIdentity) {
+        return playerIdentity.getUuid().toString().replace("-", "");
```

Comment:
Use the UUID utils: https://github.com/Minecrell/ServerListPlus/blob/master/src/main/java/net/minecrell/serverlistplus/core/util/UUIDs.java

`UUIDs.NO_DASHES.toString(...)`